apps/api/src/resources/User/resolver.ts

```diff
@@ -104,6 +106,43 @@ export class UserResolver extends ResourceResolver<resourceType>(
     return user.save();
   }
 
+  @Authorized("update:user_expiration_date")
```

Comment:
Honestly don't know how the Authorized function works, so ill need to check if this is okay